src/lib/input/input-container.spec.ts

```diff
@@ -749,6 +751,121 @@ describe('MdInputContainer', function () {
 
   });
 
+  describe('custom error behavior', () => {
+    it('should display an error message when a custom error matcher returns true', async(() => {
```

Comment:
Given that we're just using reactive form directives now, these tests don't need to be async. You should be able to safely remove the `async` and `whenStable` calls.